### PR TITLE
Cleaning up gradle files

### PR DESCRIPTION
**Pull request by markusthoemmes**
- Making gradle files even more standardized
- Moving controller/invoker command to Dockerfile/ansible playbook

### CODE REVIEW

File: core/controller/Dockerfile [hunk 1]
```diff
@@ -13,6 +13,6 @@ RUN wget --no-verbose https://github.com/swagger-api/swagger-ui/archive/v2.1.4.t
 # Copy app jars
 COPY build/distributions/controller.tar ./
 RUN tar xf controller.tar
-CMD controller/bin/controller
+CMD controller/bin/controller >> /logs/${COMPONENT_NAME}_logs.log 2>&1
```
Comment (rabbah) [inline core/controller/Dockerfile]: why dockerfile here but playbook above?

Comment (markusthoemmes) [inline core/controller/Dockerfile]: Invoker needs additional params only known at deployment time. Controller doesn't.

**Comment by rabbah** [comment]
pg?

**Comment by markusthoemmes** [comment]
PPG890

**Comment by rabbah** [comment]
Started PPG 951.

**Comment by rabbah** [comment]
Merged as https://github.com/openwhisk/openwhisk/commit/84bf14a6ed97e8255c9c15b083be5d33decaca99.

**Pull request closed without merging**